client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/RequestPermissionBottomSheet.kt

```diff
@@ -0,0 +1,76 @@
+package presentation.login.composable
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import com.beepbeep.designSystem.ui.composable.BpButton
+import com.beepbeep.designSystem.ui.composable.BpExpandableTextField
+import com.beepbeep.designSystem.ui.composable.BpTextField
+import com.beepbeep.designSystem.ui.composable.BpTransparentButton
+import com.beepbeep.designSystem.ui.theme.Theme
+import presentation.login.LoginScreenInteractionListener
+import presentation.login.LoginScreenUIState
+import resources.Resources
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun RequestPermissionBottomSheet(
+    listener: LoginScreenInteractionListener,
+    state: LoginScreenUIState,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.wrapContentHeight()
+            .padding(horizontal = Theme.dimens.space16, vertical = Theme.dimens.space24)
```

Comment:
use static spacers , don't use the dimens existed in Theme

Reply:
thanks for your review,for all  the spacers only?